web/src/pm_list.ts

```diff
@@ -16,6 +18,9 @@ let prior_dom: vdom.Tag<PMNode> | undefined;
 // This module manages the direct messages section in the upper
 // left corner of the app.  This was split out from stream_list.ts.
 
+const ls_key = "private_messages_collapsed_status";
```

Comment:
Let's use `direct_messages` in the title -- might as well save ourselves some renaming work, which is more annoying with something like this. Maybe `left_sidebar_direct_messages_collapsed_state`?

BTW, probably now is not an awful time to start doing quick side PRs for doing `direct_message` => `private_message` replacements in `web/` and `zerver/`, at least across things like variable names that don't have any API complexity to manage, so we don't have to spend human time thinking about the name change anymore.

```
tabbott@coset:~/zulip$ git grep direct_message zerver | wc
    799    3020   81698
tabbott@coset:~/zulip$ git grep private_message zerver | wc
    133     504   12159
tabbott@coset:~/zulip$ git grep private_message web/src/ | wc
    151     567   12304
tabbott@coset:~/zulip$ git grep direct_message web/src/ | wc
    186     725   17160
```


Within `zerver`, just `internal_send_private_message` is 33 of those.

Reply:
Updated to use `left_sidebar_direct_messages_collapsed_state`, I add this to my TODO.